build-logic/src/main/java/com/squareup/workflow1/buildsrc/internal/catalogs.kt

```diff
@@ -44,3 +27,48 @@ fun DependencyHandler.kotlin(
   module: String,
   version: String? = null
 ): Any = "org.jetbrains.kotlin:kotlin-$module${version?.let { ":$version" } ?: ""}"
+
+/**
+ * the jdk used in packaging
```

Comment:
Isn't this redundant w/kgx?

Reply:
I'd thought it was, but sadly no.  The catalog entry/property names are unique to individual projects.